src/ComputeSharp.D2D1/Attributes/Enums/D2D1CompileOptions.cs

```diff
@@ -176,14 +176,22 @@ public enum D2D1CompileOptions
     EnableLinking = 1 << 31,
 
     /// <summary>
-    /// The default options for shaders compiled by ComputeSharp.D2D1. Specifically, this combination
-    /// of options is the one used by precompiled shaders (ie. when using <see cref="D2DShaderProfileAttribute"/>),
-    /// and when using <see cref="D2D1PixelShader.LoadBytecode{T}()"/> or any of the overloads.
+    /// The default options for shaders compiled by ComputeSharp.D2D1. Specifically, this combination of options
+    /// is the one used by precompiled shaders (ie. when using <see cref="D2DShaderProfileAttribute"/>), and when
+    /// using <see cref="D2D1PixelShader.LoadBytecode{T}()"/> or any of the overloads, if not manually set.
+    /// </summary>
+    Default = OptimizationLevel3 | WarningsAreErrors | PackMatrixRowMajor | EnableLinking,
+
+    /// <summary>
+    /// A combination of options to aggressively optimize for maximum speed.
     /// </summary>
     /// <remarks>
-    /// This option does not include <see cref="EnableLinking"/>, which is instead automatically enabled by shaders
-    /// compiled using any of the APIs mentioned above. When manually compiling shaders from source using the APIs
-    /// from <see cref="D2D1ShaderCompiler"/> instance, <see cref="EnableLinking"/> should be manually used when needed.
+    /// This combination includes <see cref="PartialPrecision"/>, which might slightly alter the shader outputs in some cases.
```

Comment:
"which might slightly alter the shader outputs in some cases"

Actually it can alter the shader output by quite a bit in many cases! Especially for shaders that do more than a few trivial floating point operations.

For example, I had to force the precision of my color management effect graph to use Float32 because of this bug report. This screenshot was with Float16 for intermediate textures (aka effect buffer precision), so it was only rounding from FP32 to FP16 on shader boundaries, not within them, which would've made the errors way way way worse:
![image](https://github.com/Sergio0694/ComputeSharp/assets/10873410/ebbd655a-8c51-48bd-87b3-a3e0a9265226)